org.mwc.cmap.legacy/src/MWC/GUI/ICantContainLayers.java

```diff
@@ -0,0 +1,6 @@
+package MWC.GUI;
+
```

Comment:
@saulhidalgoaular - could you add a comment explaining the role of this interface?

Reply:
Sure. 1 Minute